projects/hal-components/src/lib/components/resizer/resizer.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, HostListener, Input } from '@angular/core';
+
+type direction = 'vertical' | 'horizontal';
+
+@Component({
+  selector: 'hal-resizer',
+  templateUrl: './resizer.component.html',
+  styleUrls: ['./resizer.component.scss']
+})
+export class ResizerComponent {
+  @Input() resizableElement: HTMLElement;
+  @Input() resizeDirection: direction; // set outside, and remove default
+  @Input() minHeight?: 300;
```

Comment:
hvorfor minHeight?: 300; og ikke minHeight = 300;

Reply:
Kom inn på en tråd som nevnte dette for å gjøre inputene optional, men ser jo nå at input() er optional by default. Jeg endrer til som du nevner her ;)